MaterialLib/MPL/Properties/pViscosity_IAPWS.cpp

```diff
@@ -0,0 +1,71 @@
+/**
+ * \author Norbert Grunwald
+ * \date   Sep 21, 2017
+ * \brief
+ *
+ * \copyright
+ * Copyright (c) 2012-2017, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#include "pViscosity_IAPWS.h"
+#include <cmath>
+#include "../mpComponent.h"
+
+namespace MaterialPropertyLib
+{
+ViscosityWaterIAPWS::ViscosityWaterIAPWS(Medium*) : _component(0)
+{
+    notImplemented("Viscosity_IAPWS", "Medium");
+}
+
+ViscosityWaterIAPWS::ViscosityWaterIAPWS(Phase*) : _component(0)
+{
+    notImplemented("Viscosity_IAPWS", "Phase");
+}
+
+ViscosityWaterIAPWS::ViscosityWaterIAPWS(Component* c)
+    : _component(c),
+      _H{{1.67752, 2.20462, 0.6366564, -0.241605}},
```

Comment:
It is better to move these data to cpp as local static data (according to the comments by Dima to one of my PR).  See
https://github.com/ufz/ogs/blob/master/MaterialLib/Fluid/Viscosity/WaterViscosityIAPWS.cpp#L22